### ISSUE TITLE
radio-button: Chromevox and JAWS screen readers do not announce the label of the radio buttons

### ISSUE THREAD

**Issue by kittenish** [angular/material#11974]
## Bug

# Demo and steps to reproduce the issue
[This blank StackBlitz demo](https://stackblitz.com/edit/angularjs-material-blank?file=app%2Fapp.template.html) can be used to create a reproduction that demonstrates your issue.

**Demo URL (required)***:  https://codepen.io/jiaruigao/pen/VweGJrp?editable=true%3Dhttps%3A%2F%2Fmaterial.angularjs.org%2FHEAD%2Fdemo%2FradioButton

#### Detailed Reproduction Steps
1. Enable Chromevox or JAWS screen readers
2. Do not pre-select any radio button within a radio group
3. Navigate to a radio group using keyboard tab key
4. Observe the first radio button is focused
5. Observe Chromevox and JAWS screen readers do not announce the label of the radio buttons at all.

### Explain the expected behavior
In step 5, the focused button should be announced.

### Explain the current behavior
In step 5, the focused button is not announced.

### Discuss the use-case or motivation for changing the existing behavior
Screen readers should announce radio button when it's focused.

### List the affected versions of AngularJS, Material, OS, and browsers

- AngularJS: 1.
- AngularJS Material: 1.
- Chromevox - Chrome browser - Google Chrome OS Version 81.0.4044.103
- JAWS Version 2020.2001.70 ILM - Chrome Version 81.0.4044.138 - Windows 10 Enterprise (64-bit)

#### Screenshots
Current behavior with Chromevox:
<img width="1548" alt="Screen Shot 2020-07-22 at 3 10 45 PM" src="https://user-images.githubusercontent.com/18439611/88224265-57186580-cc2e-11ea-8982-f3b5e4338a2b.png">
